lib/model/cardinalPortPosition.js

```diff
@@ -15,6 +17,7 @@ var CardinalPortPosition = function(direction,percentage) {
  * @function getDirection
  */
 CardinalPortPosition.prototype.getDirection = function() {
+  "use strict";
```

Comment:
For future reference, the thing I mentioned in the above comment is possible; browserify will encapsulate modules in such a way that we can put `"use strict";` at file level scope without causing any problems with the global scope. Alternatively, we can use a plugin like [this](https://www.npmjs.com/package/strictify) and not have to worry about it at all.